_data/nikhiltripathi1/nikhiltripathi1.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Nikhil Tripathi # No longer than 28 characters
+institution: Dr. Akhilesh Das gupta Institute of Technology and Management # no longer than 58 characters
+profile_pic: nikhil.png # Name and extension of your profile picture(ex. mona.png)
+quote: exploring or escaping, what should we choose? # no longer than 100 characters
+github_user: nikhiltripathi1
+---
```

Comment:
```suggestion
name: Nikhil Tripathi
institution: Dr. Akhilesh Das gupta Institute of Technology and Management
profile_pic: nikhil.png
quote: exploring or escaping, what should we choose?
github_user: nikhiltripathi1
---
```

Reply:
I think having those comments in the markdown file is fine though. Most of them already got approved and pushed to the repository.